### PR TITLE
US 6

### PR DESCRIPTION
**Pull request by gcude**
Pull request for US-6 Tasks 26, 27, and 29 which add Driver classes and GUI elements.

### CODE REVIEW

File: src/main/java/memoranda/ui/WorkPanel.java [hunk 1]
```diff
@@ -227,9 +275,23 @@ else if (pan.equals("EVENTS"))
...
+				driverB_actionPerformed(null);	//3/27/2018 GC Added for DriverPanel
 		}
 	}
+	
+	/*
+	 * Added 3/27/2018 GC
+	 * For DriverPanel
+	 */
+	public void driverB_actionPerformed(ActionEvent e) {
+		System.out.println("Here next");
```
Comment (cpremuda) [inline src/main/java/memoranda/ui/WorkPanel.java]: Did we mean to leave this in?

File: src/main/java/memoranda/ui/WorkPanel.java [hunk 2]
```diff
@@ -200,16 +210,54 @@ public void actionPerformed(ActionEvent e) {
...
+		driverB.setBorderPainted(false);
+		driverB.setContentAreaFilled(false);
+		driverB.setFocusPainted(false);
+		driverB.setHorizontalTextPosition(SwingConstants.CENTER);
+		driverB.setText(Local.getString("Drivers"));
+		driverB.setVerticalAlignment(SwingConstants.TOP);
+		driverB.setVerticalTextPosition(SwingConstants.BOTTOM);
+		driverB.addActionListener(new java.awt.event.ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				System.out.println("Here");
```
Comment (cpremuda) [inline src/main/java/memoranda/ui/WorkPanel.java]: Maybe another one left in by accident

File: src/main/java/memoranda/ui/WorkPanel.java (hunk 1 above)
Comment (gcude) [reply src/main/java/memoranda/ui/WorkPanel.java]: Removed

File: src/main/java/memoranda/ui/WorkPanel.java (hunk 2 above)
Comment (gcude) [reply src/main/java/memoranda/ui/WorkPanel.java]: Removed Thanks!

**Pull request merged**